datasets/ladi.yaml

```diff
@@ -0,0 +1,54 @@
+Name: Low Altitude Disaster Imagery (LADI) Dataset
+Description: The Low Altitude Disaster Imagery (LADI) Dataset consists of human and machine annotated airborne images collected by the Civil Air Patrol in support of various disaster responses from 2015-2019. The initial release of LADI focuses on the Atlantic hurricane seasons and coastal states along the Atlantic Ocean and Gulf of Mexico. Annotations are included for major hurricanes of Harvey, Maria, and Florence. Two key distinctions are the low altitude, oblique perspective of the imagery and disaster-related features, which are rarely featured in computer vision benchmarks and datasets.
+Documentation: https://github.com/LADI-Dataset/ladi-overview/README.md
+Contact:  ladi-dataset-admin@mit.edu
+ManagedBy: "[MIT Beaver Works](https://beaverworks.ll.mit.edu/)"
+UpdateFrequency: Periodically
+License: Creative Commons 4.0 (CY4.0)
```

Comment:
Double presence of license field, I think you can remove this one.

Reply:
Removed in commit 561feea